zstacklib/zstacklib/gpu/vendors/nvidia.py

```
@@ -353,6 +354,131 @@ def pre_detach_from_host(cls):
+                # nvidia-smi uses 8-digit domain (e.g. 00000000:xx:xx.x),
+                # normalize to 4-digit (0000:xx:xx.x) for comparison
+                domain = bus_id.split(":")[0]
+                if len(domain) == 8:
+                    bus_id = bus_id[4:]
+                if bus_id == target:
+                    dev_path = "/dev/nvidia%s" % idx
+                    if os.path.exists(dev_path):
+                        return dev_path
+                    return None
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)